tests/mitol/payment_gateway/test_cybersource.py

```diff
@@ -167,3 +172,40 @@ def __init__(self, data, method):
+    from test payments. (These have fake keys in them and won't pass the
+    signaure validation, so this test doesn't validate them.)
+    """
+
+    test_success_payload = json.loads(
+        '{"req_locale": "en-us","req_card_type_selection_indicator": "1","auth_trans_ref_no": "79154224TYSHOGMT","req_item_0_sku": "57-1","req_item_0_code": "57","req_bill_to_surname": "Test","req_item_0_unit_price": "100.00","req_item_0_quantity": "1","req_card_expiry_date": "02-2023","req_bill_to_phone": "9015551212","card_type_name": "Visa","auth_amount": "100.00","auth_response": "100","bill_trans_ref_no": "79154224TYSHOGMT","req_payment_method": "card","auth_time": "2022-01-28T210125Z","transaction_id": "6434036856426521704004","req_card_type": "001","req_override_custom_receipt_page": "http://mitxonline.odl.local:8013/ecommerce-test/checkout_decode_response","req_consumer_id": "admin","auth_avs_code": "X","auth_code": "888888","req_bill_to_address_country": "US","req_profile_id": "not an actual profiie ID","req_override_custom_cancel_page": "http://mitxonline.odl.local:8013/ecommerce-test/checkout_decode_response","signed_date_time": "2022-01-28T21:01:25Z","req_bill_to_address_line1": "123 Any Street","req_card_number": "xxxxxxxxxxxx4242","signature": "4Jd/nL6fwoHP0dHcndY+QzcllbZ5ERaLCb7si5hZlX0=","req_bill_to_address_city": "Memphuis","req_bill_to_address_postal_code": "38104","reason_code": "100","req_bill_to_forename": "Test","request_token": "Axj//wSTXMoyv+6BzDZEABEg3csWrRkyaVLNORPjzaiiOMmFiGgFRHGTCxDWkDpxAfshk0ky9GLJu+4MGEmuZRlf90DmGyIA7ieV","req_amount": "100.00","req_bill_to_email": "test@test.local","auth_avs_code_raw": "I1","req_currency": "USD","decision": "ACCEPT","req_item_0_tax_amount": "0","req_customer_ip_address": "172.21.0.1","message": "Request was processed successfully.","signed_field_names": "transaction_id,decision,req_access_key,req_profile_id,req_transaction_uuid,req_transaction_type,req_reference_number,req_amount,req_currency,req_line_item_count,req_locale,req_payment_method,req_override_custom_receipt_page,req_override_custom_cancel_page,req_consumer_id,req_item_0_code,req_item_0_name,req_item_0_quantity,req_item_0_sku,req_item_0_tax_amount,req_item_0_unit_price,req_bill_to_forename,req_bill_to_surname,req_bill_to_email,req_bill_to_phone,req_bill_to_address_line1,req_bill_to_address_city,req_bill_to_address_state,req_bill_to_address_country,req_bill_to_address_postal_code,req_card_number,req_card_type,req_card_type_selection_indicator,req_card_expiry_date,card_type_name,req_customer_ip_address,message,reason_code,auth_avs_code,auth_avs_code_raw,auth_response,auth_amount,auth_code,auth_trans_ref_no,auth_time,request_token,bill_trans_ref_no,signed_field_names,signed_date_time","req_transaction_uuid": "d0c0b45be7974460b7bbf49db8a27f14","req_transaction_type": "sale","req_access_key": "not a real access key","req_item_0_name": "Test Item 0","req_reference_number": "14","req_bill_to_address_state": "TN","req_line_item_count": "1"}'
```

Comment:
GitGuardian is flagging this block, is there a secret value in here that should be redacted (even if it's a dev one)? If not, you can add a GitGuardian config to ignore it: https://github.com/GitGuardian/ggshield/blob/main/.gitguardian.example.yml